source/backend/routes/users.js

```diff
@@ -3,6 +3,36 @@ const usersModel = require('../database/models/usersModel');
 
 const router = express.Router();
 
+/* GET users/completedRecipes */
+router.get('/completedRecipes', async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const completedRecipesList = await usersModel.getCompletedRecipes(userId);
+    res.send(completedRecipesList);
```

Comment:
we would need a return statement here, and the send function default status to 304 meaning not modified. By design we would want this get route to return a 200 being the success fetch.
Syntax:
`return res.status(200).json(completedRecipesList);`

Reply:
forgot to change that. thanks